types/cloneable-readable/cloneable-readable-tests.ts

```diff
@@ -1,5 +1,5 @@
-import { PassThrough } from 'stream';
-import cloneable = require('cloneable-readable');
+import { PassThrough } from "stream";
+import cloneable = require("cloneable-readable");
```

Comment:
Let's keep the quotes in the same style they were. I don't think this matters, but it adds unnecessary churn in this commit

Reply:
My linter auto-corrected this, but let me revert.